packages/fxa-auth-server/fxa-oauth-server/lib/grant.js

```diff
@@ -161,6 +166,11 @@ module.exports.generateTokens = async function generateTokens(grant) {
     result.id_token = await generateIdToken(grant, result.access_token);
   }
 
+  if (grant.scope && grant.scope.contains(OAUTH_SCOPE_SESSION_TOKEN)) {
+    result.session_token =
```

Comment:
Nit: You're not returning a fully-functional sessionToken here, you're only returning the id. I suggest naming this field `session_token_id` or `session_id` to avoid confusion.